janitor/functions.py

```diff
@@ -3869,3 +3813,230 @@ def sort_naturally(
     """
     new_order = index_natsorted(df[column_name], **natsorted_kwargs)
     return df.iloc[new_order, :]
+
+
+@pf.register_dataframe_method
+def expand_grid(df: pd.DataFrame = None,
```

Comment:
Smart! This allows us to both chain it and use it as a function that is _not_ attached to a DataFrame too.